__tests__/app.test.js

```diff
@@ -94,3 +94,48 @@ describe("GET /api", () => {
         });
     });
 })
+
+describe("GET /api/reviews", () => {
+    test.only("status:200, responds with an array of all the review objects with the correct properties, sorted in descending order", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then((response) => {
+            console.log(response.body.reviews)
+            response.body.reviews.forEach((review) => {
+                expect(typeof review.owner).toBe('string');
+                expect(typeof review.title).toBe('string');
+                expect(typeof review.review_id).toBe('number');
+                expect(typeof review.category).toBe('string');
+                expect(typeof review.review_img_url).toBe('string');
+                expect(typeof review.created_at).toBe('string');
+                expect(typeof review.votes).toBe('number');
+                expect(typeof review.designer).toBe('string');
+                expect(typeof review.comment_count).toBe('string');
+            })
+        expect(response.body.reviews).toBeSortedBy('created_at', {descending: true})
```

Comment:
Really nice use of jest-sorted! I think this could be a separate test to distinguish between behaviours🙂